pages/dashboard.tsx

```diff
@@ -1,17 +1,48 @@
 import type { NextPage } from "next";
 import Head from "next/head";
+
+import Header from "components/shared/Header";
+
 import Navbar from "components/shared/Navbar";
 import Footer from "components/shared/Footer";
 import { PoolList } from "components/dashboard/PoolList";
 import { ShareList } from "components/dashboard/ShareList";
-import { ActionButtons } from "#/components/dashboard/ActionButtons";
+import { PlusSmallIcon } from "@heroicons/react/20/solid";
+import Link from "next/link";
+
+const secondaryNavigation = [
+  { name: "Last 7 days", href: "#", current: true },
+  { name: "Last 30 days", href: "#", current: false },
+  { name: "All-time", href: "#", current: false },
+];
+const stats = [
```

Comment:
hardcoded?
TODO: replace with actual values

Reply:
u fixed that :)